examples/worlds/actor.sdf

```diff
@@ -6,20 +6,20 @@
   <world name="actors">
     <plugin
       filename="ignition-gazebo-physics-system"
-      name="ignition::gazebo::systems::Physics">
+      name="gz::sim::systems::Physics">
```

Comment:
I'm still compiling this, so just leaving a note here that we should check if `ignition::gazebo` still works.

Reply:
Ok, I just tested it and it doesn't work. But the fix is simple, we need to add aliases like this to every system plugin:

```diff
diff --git a/src/systems/physics/Physics.cc b/src/systems/physics/Physics.cc
index b9df0909b..451773597 100644
--- a/src/systems/physics/Physics.cc
+++ b/src/systems/physics/Physics.cc
@@ -3604,3 +3604,5 @@ IGNITION_ADD_PLUGIN(Physics,
                     Physics::ISystemUpdate)
 
 IGNITION_ADD_PLUGIN_ALIAS(Physics, "gz::sim::systems::Physics")
+// Deprecated, remove on version 8
+IGNITION_ADD_PLUGIN_ALIAS(Physics, "ignition::gazebo::systems::Physics")
```

You can check it worked either by changing a world to load the `ignition::gazebo` version of the plugin, or with the `ign plugin` CLI:

```
$ ign plugin -i -v -p install/lib/libignition-gazebo7-physics-system.so
Loading plugin library file [install/lib/libignition-gazebo7-physics-system.so]
Loader State
        Known Interfaces: 4
                gz::sim::v7::ISystemUpdate
                gz::sim::v7::System
                gz::sim::v7::ISystemReset
                gz::sim::v7::ISystemConfigure
        Known Plugins: 1
                [gz::sim::v7::systems::Physics]
                        has 2 aliases:
                                [gz::sim::systems::Physics]
                                [ignition::gazebo::systems::Physics]
                        implements 4 interfaces:
                                gz::sim::v7::ISystemConfigure
                                gz::sim::v7::ISystemReset
                                gz::sim::v7::ISystemUpdate
                                gz::sim::v7::System
```

Note the alias

Reply:
https://github.com/gazebosim/gz-sim/pull/1496/commits/5c21f048222d0d0c7853de91fd5584c60a44d2e9
:>

The specific steps I did are in the commit message